mappings/net/minecraft/util/math/Direction.mapping

```diff
@@ -28,6 +28,11 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_10143 byId (I)Lnet/minecraft/class_2350;
 		ARG 0 id
 	METHOD method_10144 asRotation ()F
+	METHOD method_10145 listClosest (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
+		COMMENT Helper function that returns the 3 directions given, followed by the 3 given in opposite order.
```

Comment:
```suggestion
		COMMENT Helper function that returns the 3 directions given, followed by the 3 opposite given in opposite order.
```
Think that's how it actually is, or is it?

Reply:
Yes you're right, forgot a word